pkg/app/app.go

```diff
@@ -289,6 +289,24 @@ func (a *App) Lint(c LintConfigProvider) error {
 	}, SetFilter(true))
 }
 
+func (a *App) Pull(c PullConfigProvider) error {
```

Comment:
Can we leave the name `pull` for more broader use and use e.g. a more specific variant like `prepare-charts` for now? If you prefer shorter, I think `prepare` is also fine as we already use `prepare` with the exact same meaning in our [hooks](https://github.com/roboll/helmfile#hooks)

Reply:
how about `fetch`? to be in line with helm :-)

`helm fetch` is alias of `helm pull`

Reply:
@itscaro Hey! Yeah- maybe good enough?

What scares me is that it isn't exactly just `fetch` either- `PrepareChart` involves the nasty "chartify" step that runs various commands to turn even local directory containing k8s manifests/kustomize into a temporary helm chart.

https://github.com/roboll/helmfile/blob/25baebe2b18cc059389b4bb1fcd334383466f0e2/pkg/state/state.go#L1051

WDYT? Would it be okay to just call it `helmfile fetch`?

Reply:
Yes I did the change to call it `helmfile fetch` and yes local chart will be copied into the temporary directory too. The current behaviour of this command is a bit intutive, I leave the default output directory as the temporary directory which is deleted afterwards. To keep downloaded charts (or locally copied ones) `--output-dir` must be set. Tell me what do you think?

Reply:
Thanks! It would look perfect if `--output-dir` defaults to somewhere under `$HOME/.helmfile/cache`, as other Helmfile commands fetches charts into there. That way,  running `helmfile fetch` beforehand would probably make subsequent `helmfile apply` faster.